koku/masu/external/downloader/gcp_local/gcp_local_report_downloader.py

```diff
@@ -30,52 +26,6 @@ class GCPReportDownloaderError(Exception):
     pass
 
 
-def create_daily_archives(tracing_id, account, provider_uuid, filename, filepath, manifest_id, start_date, context):
-    """
-    Create daily CSVs from incoming report and archive to S3.
-
-    Args:
```

Comment:
Why is this function being removed from the local gcp downloader?

Reply:
This was me cleaning up the local downloader and moving to the standard non local one, just reducing some code. I forgot I had this change tbh :sweat_smile: not a terrible idea if nothing breaks since its only local but we can drop it out if we want :+1: